superset/db_engine_specs/bigquery.py

```diff
@@ -38,6 +43,28 @@
     + "permission in project (?P<project>.+?)"
 )
 
+ma_plugin = MarshmallowPlugin()
+
+
+class EncryptedField(fields.String):
+    pass
+
+
+class BigQueryParametersSchema(Schema):
+    credentials_info = EncryptedField(description="credentials.json file for BigQuery")
+
+
+def encrypted_field_properties(self, field: Any, **_) -> Dict[str, Any]:  # type: ignore
+    ret = {}
+    if isinstance(field, EncryptedField):
+        if self.openapi_version.major > 2:
+            ret["x-encrypted-extra"] = True
+    return ret
+
+
+class BigQueryParametersType(TypedDict):
+    pass
```

Comment:
This now has `credentials_info`

```suggestion
    credentials_info: Dict[str, Any]
```